aidants_connect_habilitation/tests/test_admin.py

```diff
@@ -19,6 +19,13 @@
 
 @tag("admin")
 class OrganisationRequestAdminTests(TestCase):
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+        cls.org_request_admin = OrganisationRequestAdmin(
+            OrganisationRequest, AdminSite()
+        )
```

Comment:
Ici, je pense que le problème est lié [à une toute petit fonctionnalité supprimée dans Django 4.1](https://docs.djangoproject.com/en/4.1/releases/4.1/#features-removed-in-4-1) :

> Support for assigning objects which don’t support creating deep copies with `copy.deepcopy()` to class attributes in `TestCase.setUpTestData()` is removed.

Concrètement, les objets `ModelAdmin` ne peuvent pas être récursivement clonés avec `copy.deepcopy()` donc on ne peut plus les créer dans `setUpTestData`.

C'est vraiment pas grave mais faut le savoir, d'autant que ça génère une erreur qui n'est vraiment pas intuitive.